src/util/componentName.js

```diff
@@ -0,0 +1,8 @@
+export function makeComponentName({componentType, componentId}) {
```

Comment:
@YiningChen can you share the reasoning for having this? In particular, why not just make the `{componentType, componentId}` structure the thing we use universally to describe a UI component, rather than still using the string representation in some cases?

Reply:
No good reasoning! I assumed there was already a reason it was a string, which is a bad reason xD Will change, thanks!